BOUNTY.md

```diff
@@ -0,0 +1,115 @@
+# GraphQL.NET Bounty Program
+
+The GraphQL.NET team has worked to create a program where community developers can work on specific new features
+of GraphQL.NET for compensation. These features are determined by GraphQL.NET maintainers to have the
+potential for high-impact improvement to the product. Design proposals that would fulfill a bounty may
+be made either by individuals or teams of developers, but the final payout will be made to only one stakeholder.
+
+
+## Bounty Process Flowchart
+
+1. The GraphQL.NET team publishes a Request For Proposals (RFP) describing a feature that the team has decided
+   should be prioritized, or a bug that is in need of being researched and fixed.
+    * Initial Bounty Status set to **Accepting Proposals**
+2. Developers write proposals for how they would implement the feature or fix the bug described in the RFP.
+3. The GraphQL.NET team works with developers to clarify and refine their proposals.
+4. A proposal is accepted. The developer(s) responsible for the proposal work to implement the feature or
+   bug fix according to their proposal.
+    * Bounty status changed to **In Progress**
+5. Pull requests are submitted to implement the proposal by the developers.
+    * Bounty status changed to **Under Review**
+6. Pull requests are reviewed by project maintainers, and the pull request developers make any changes to the
+   pull requests accordingly.
+7. Pull requests are merged into the master or develop branch.
+8. Developer(s) submit invoice to the [GraphQL.NET Open Collective](https://opencollective.com/graphql-net)
+   for bounty amount, confirmed by GraphQL.NET team members.
+    * Bounty status changed to **Claimed**
+9. Payment disbursed to developer(s) by the Open Source Collective from GraphQL.NET funds.
```

Comment:
```suggestion
9. Payment disbursed to developer(s) by the Open Source Collective from GraphQL.NET funds.
    * Bounty status of issue changed to **Paid**
```
?

Reply:
Do we need a confirmation about payment from the PR author?

Reply:
Not sure exactly what you mean.  I'm sure us maintainers will handle the payment or the PR author will inform us if we forget!  The badge/label is really so we can indicate the number of paid bounties within the readme easily.

Reply:
I mean the fact of transferring money to the author's account.

Reply:
OpenCollective is responsible for the actual payment.  joemcbride just needs to approve the invoice.  There's nothing additional that we need to write here.  I'm sure if the author has an issue they will bring it up.

Reply:
OK. I expected such an answer and only wanted to clarify.